anchore_engine/configuration/localconfig.py

```diff
@@ -55,7 +55,7 @@
         },
     },
     "policy_bundles_dir": "bundles/",
-    "max_compressed_image_size": None,
+    "max_compressed_image_size": -1,
```

Comment:
Also, here if we can add unit since it isn't bytes: max_compressed_image_size_mb